how-to.md

```diff
@@ -107,9 +107,21 @@ You can [preview your post while you write it](#how-to-preview-your-post-locally
 This site is built around [**Jekyll**](https://jekyllrb.com/). Jekyll takes all the markdown files and generates a static html website.
 
 1.  [Install Ruby using rbenv](/how-to-install-ruby). Don't use `apt-get` since its version of Ruby is too old.
-2.  Install Jekyll by running : `gem install bundler jekyll`.
+2.  Install `bundler` by running : `gem install bundler`.
```

Comment:
I think to ensure we install the same version of bundler, we should use `gem install bundler:2.1.4` here.

Reply:
Yes, good catch